Assets/MRTK/Core/Providers/InputSimulation/BaseInputSimulationService.cs

```diff
@@ -130,6 +140,12 @@ protected BaseController GetOrAddControllerDevice(Handedness handedness, Control
                     controller = new SimulatedArticulatedHand(TrackingState.Tracked, handedness, inputSource);
                     controllerType = typeof(SimulatedArticulatedHand);
                     break;
+                case ControllerSimulationMode.MotionController:
+                    st = SupportedControllerType.WindowsMixedReality;
+                    inputSource = Service?.RequestNewGenericInputSource($"{handedness} MotionController", RequestPointers(st, handedness), InputSourceType.Controller);
```

Comment:
Is it possible from the name to understand that this is a simulated motion controller? I'm thinking if people are looking at the list of active controllers in the scene view, they might be confused when they see a motion controller but it is not mentioned that it is simulated.

Reply:
I don't think people can tell from just the name. The naming was based on the current behavior when a new simulated hand is created (i.e. in line 139 of the same file.) I am happy to add the term "simulated" to both if you wish.

Reply:
How does input binding work on device? By default the actions are bound to keyboard keys, do you need a physical keyboard on device to simulate hands? What about mouse movements? If that somehow works for hands i would expect it to work for motion controllers as well, the code is very similar.